tools/cpplint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/env python
```

Comment:
I don't think `python2.4` would work much better :) Just pulled the same convention as what we have in `configure` and `tools/test.py` -- would personally have gone with `python2` but +1 for consistency.